x-pack/plugins/rule_registry/server/plugin.ts

```diff
@@ -4,19 +4,33 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
-import { PluginInitializerContext, Plugin, CoreSetup, Logger } from 'src/core/server';
+import {
+  PluginInitializerContext,
+  Plugin,
+  CoreSetup,
+  Logger,
+  KibanaRequest,
+  CoreStart,
+  IContextProvider,
+} from 'src/core/server';
+import { SecurityPluginSetup } from '../../security/server';
+import { AlertsClientFactory } from './alert_data_client/alerts_client_factory';
+import { PluginStartContract as AlertingStart } from '../../alerting/server';
+import { RacApiRequestHandlerContext, RacRequestHandlerContext } from './types';
+import { defineRoutes } from './routes';
 import { SpacesPluginStart } from '../../spaces/server';
 
 import { RuleRegistryPluginConfig } from './config';
 import { RuleDataPluginService } from './rule_data_plugin_service';
 import { EventLogService, IEventLogService } from './event_log';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface RuleRegistryPluginSetupDependencies {}
+export interface RuleRegistryPluginSetupDependencies {
+  security: SecurityPluginSetup;
```

Comment:
Nit, this is an optional dependency

```suggestion
  security?: SecurityPluginSetup;
```